channels/us.m3u

```diff
@@ -929,9 +931,13 @@ https://na-all23.secdn.net/pegstream3-live/play/65ea794b-dd82-41ce-8e98-a9177289
 https://a.jsrdn.com/broadcast/4hhfi556/indexR720P.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/tbcqI3o.jpg" group-title="Sport",US SPORT Tennis Plus 1
 https://playoutengine.sinclairstoryline.com/playout/f2f8b269-dd85-4434-bdd3-b3a64ca9cd60/g.m3u8
+<<<<<<< HEAD
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/FzxuNhl.png" group-title="Local",KVVB-TV 33
+=======
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="" group-title="",USA Network
 https://cdn008.teleuptv.net/JnVLClgyvD/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/FzxuNhl.png" group-title="Local",Victorville | KVVB-TV 33
+>>>>>>> d5a680156a1edfcb7e12489ef290e653deba9b5e
```

Comment:
this line must be deleted

Reply:
That's because I deleted the City name before the actual Channel name I suppose.